### ISSUE TITLE
generator not supported?

### ISSUE THREAD

**Issue by satlxq** [chinchang/web-maker#159]
js preprocesser ES6(Babel);

```javascript
function* gen(x){
    var y = yield x+1;
    return y;
}
var g = gen(3);
var k = g.next();
console.log(k.value)
```
getting error in console panel

> ReferenceError: regeneratorRuntime is not defined
    at script 3:25 

I got this error code line in chrome Devtool source panel
```javascript
var _marked = [gen].map(regeneratorRuntime.mark);
```

**Comment by chinchang** [comment]
@satlxq I suppose you are on **Babel** mode and it seems, Babel isn't able to compile that code. I'll check that but meanwhile you can just run your code on **JS** mode as generators are available natively in Chrome.

**Comment by satlxq** [comment]
@chinchang  thanks, It works after switching to JS mode

**Comment by chinchang** [comment]
FYI, it will get fixed for Babel also in v2.8.0

**Comment by chinchang** [comment]
@satlxq Generators will work with ES6 also now in v2.8.0
